sim/core/buffs.go

```diff
@@ -772,23 +772,29 @@ func registerUnholyFrenzyCD(agent Agent, numUnholyFrenzy int32) {
 		numUnholyFrenzy)
 }
 
-func UnholyFrenzyAura(character *Unit, actionTag int32) *Aura {
+func UnholyFrenzyAura(agent Agent, actionTag int32) *Aura {
 	actionID := ActionID{SpellID: 49016, Tag: actionTag}
 
+	character := agent.GetCharacter()
+
+	// If this character has an effect that multiplies enrage effects, incorporate it
+	// Mainly for Fury Warrior's Unshackled Rage mastery
+	effect := 1.0 + (0.2 * character.PseudoStats.EnragedEffectMultiplier)
```

Comment:
SimC implemented this interaction but it's not clear looking at the mastery's spell effects if it would or not. I'll revert this and leave a note to test it